File: 251010/A 부터 B 까지/a-to-b.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a, b;
    cin >> a >> b;
    cout << a << " ";

    while(true) {
        if(a % 2 == 0) {
            a += 3;

            if(a > b) {
                break;
            }

            cout << a << " ";
        } else {
            a = a*2;

            if(a > b) {
                break;
            }

            cout << a << " ";
        }

    }
    return 0;
}

File: 251028/A 부터 B 까지/a-to-b.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        string[] input = Console.ReadLine().Split();
        int a = int.Parse(input[0]);
        int b = int.Parse(input[1]);

        while(a <= b) {
            if(a % 2 == 0) {
                Console.Write(a + " ");
                a += 3;
            } else if(a % 2 == 1) {
                Console.Write(a + " ");
                a *= 2;
            }
        }
    }
}
